fuzzydir.lua

```diff
@@ -87,9 +87,11 @@ function add_all(to, from)
     end
 end
 
+local is_windows = package.config:sub(1, 1) == "\\"
+
 function contains(t, e)
     for index, element in pairs(t) do
-        if element == e then
+        if is_windows and element:lower() == e:lower() or element == e then
```

Comment:
`contains` is meant to be a "general-purpose" function, so it's not supposed to include any custom logic like this one.

I'd say that the better solution would be to just convert all paths to lowercase (on Windows only) in the `normalize` function. Basically, something like this:
```lua
if is_windows then
    path = path:lower()
end
```

Reply:
Makes sense. Done.